### DISCUSSION TITLE
Would love if the contrast on the currently selected file is increased in the project view bar on the left

### DISCUSSION THREAD

**Discussion by abovestratosphere** [one-dark/jetbrains-one-dark-theme#297] [General]
I think most folks have the "Always select opened file" options ticked in the project view bar, so it always jumps around to your currently opened file. But I think the contrast is a bit too low to the point where I can't tell which file I have open quickly and I have to really look for it.

Here, I'm using GoLand. The green is test files, the light grey is generated files, the black is regular go files, and the super hard to spot medium-greyish color in the middle is my currently selected file. I know I can probably tweak this myself but if more people feel the same I think it'd be good to increase the contrast on this. I think it can even go as far as a bright blue or something that can be spotted instantly. This is apparently supposed to be a [supported scope](https://www.jetbrains.com/help/idea/configuring-scopes-and-file-colors.html#predefined-scopes-list) and should be editable via the "File Colors" setting, but the "Current file" scope is not there for me. 
![image](https://user-images.githubusercontent.com/11040393/218293316-7e0075a2-524d-4841-b574-3afacfe7ce58.png)
Overall still super satisfied with this theme.

**Comment by Unthrottled** [comment]
Hmm, this is an interesting problem.

>  I think it can even go as far as a bright blue or something that can be spotted instantly.

The issue with that is the point of the brighter color is to know whether or not the window is in focus. Having a brighter color would indicate to the user that the tree has focus and you can navigate in it. That is why the "inactive" colors are more muted, just enough to see them but not enough to indicate focus. 

<img width="602" alt="Screenshot 2023-02-12 at 1 01 48 PM" src="https://user-images.githubusercontent.com/15972415/218331485-9a537cea-7ffd-4c18-b913-40d321dfdbd3.png">

The problem is that you have issues with the `Grey` named color, as it aligns pretty much the same. The named colors are used in other places, not just for denoting file scopes, but you can color directories and other things.

So I could change the inactive color to be a muted `blue`, but that conflicts with the `Blue` named color. Then again I wonder how much `Blue` is used. `Yellow` & `Green` are the defaults. 

Let me think about it some more, and see if there is a good solution that still matches the theme but has overall good usability for any monitor set up. Thanks!

**Reply by abovestratosphere** [reply]
Appreciate you looking into this. As a side note, do you have any idea why the "currently selected file" is not a scope in the File Colors settings dialogue? If that was a scope I can change this for myself in 2 seconds. According to JetBrains' docs it should be one of the predefined scopes. In fact, many of the predefined scopes are missing in this menu, and they make no mentions of this in the documentation. 
![image](https://user-images.githubusercontent.com/11040393/218348556-2542d379-7b8a-4188-a6c8-56dceda51323.png)

**Reply by Unthrottled** [reply]
If you are referring to the color that doesn't contrast well, that is defined by the UI look & feel and not customizable via settings. It is themeable though.

Key: https://github.com/JetBrains/intellij-community/blob/master/platform/platform-resources/src/themes/metadata/JDK.themeMetadata.json#L1385
In the project: https://github.com/one-dark/jetbrains-one-dark-theme/blob/67f5ba3326b67c7d6f49575f3a196bc553fbe8bf/buildSrc/templates/oneDark.template.theme.json#L33
(is currently the catch all, so you might want to be explicit for your custom version)
